front/app/components/IdeaCards/messages.ts

```diff
@@ -10,10 +10,6 @@ export default defineMessages({
     defaultMessage:
       'No results found. Please try a different filter or search term.',
   },
-  filteredResultsFound: {
-    id: 'app.containers.IdeaCards.filteredResultsFound',
-    defaultMessage: 'New search results have loaded.',
```

Comment:
The copy seems to have been added, but wasn't used in the previous solution for this.